manifests/profile/cache_rpms.pp

```diff
@@ -1,7 +1,11 @@
-class bootstrap::profile::cache_rpms {
-  $pkglist = template('bootstrap/packages.erb')
+class bootstrap::profile::cache_rpms (
```

Comment:
yes, although I think it really shouldn't be totally idempotent because we want to check for updates.